src/pages/jobs/2020-07-31-test-job/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+layout: job
+title: 'Cat Herder @ Epos Now'

```

Comment:
Gonna probably need to add the "Generic" spoken about last night, it seems like the approach would be to give `@static-fns` the Type you expect in return, rather than any type be specified within that library. Allowing the website to specify it's own types.

However, note here to say about the fields proposed:

- Role Tag. e.g. Frontend, Backend, Full Stack, Mobile, Software, Data Science
- Salary
- Company
- Location
- Seniority
- ~~Expiry Date~~